pyproject.toml

```diff
@@ -52,12 +52,12 @@ pytest-shared-session-scope = "^0.4.0"
 azure-identity = "^1.23.0"
 azure-core = "^1.34.0"
 requests = "^2.32.4"
-azure-mgmt-sql = "^3.0.1"
+azure-mgmt-sql = "^4.0.0b21"
 pyodbc = "^5.0.1"
 azure-monitor-query = "^1.2.0"
-azure-mgmt-monitor = "^6.0.2"
+azure-mgmt-monitor = "^7.0.0b1"
 azure-mgmt-alertsmanagement = "^1.0.0"
-azure-mgmt-resource = "^24.0.0"
+azure-mgmt-resource = "^23.3.0"
```

Comment:
This is not a bump, but downgrade to match the version used in azure-cli.
I know downgrading a package without a specific reason is not good, but I think it's ok since azure-cli officially use 23.3.0 as well.